global.json

```diff
@@ -8,7 +8,7 @@
     "dotnet": "9.0.100-alpha.1.23615.4"
```

Comment:
Downgrade the SDK to 8.0.101 (also in L3).

Reply:
Do I do it manually or is there a command to automate the downgrade?

Reply:
What is L3?

Reply:
oh, _line 3_.

Reply:
manually